openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -484,6 +484,10 @@ def post(self, request):
                 address already exists
             HttpResponse: 403 operation not allowed
         """
+        if settings.FEATURES.get('ENABLE_THIRD_PARTY_ONLY_AUTH') and not pipeline.running(request):
```

Comment:
Are there accompanying UI changes to prevent displaying the username and password fields at login and registration time?  Otherwise, as it currently stands, the user is going to get an unfriendly message only after a round-trip with the backend.

Reply:
No UI changes in this PR. Let me know what do you suggest for it?

Reply:
In my opinion, the entire form needs to remove rather than the only fields. What's your opinion on that?

Reply:
@blarghmatey can you remind me what we did about the UI here?

Reply:
In our theme we removed the username and password fields on the login page, but on the registration page we had to hide the form with CSS, because the account creation process requires filling out and submitting the form with Javascript in order to function. When we had removed the registration form entirely it broke account creation.